arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -847,20 +834,43 @@
         }
       },
       "required": [
-        "storageAccountName"
+        "storageAccountName",
+        "protectedAccountKeyName",
+        "blobEndpoint",
+        "queueEndpoint",
+        "tableEndpoint"
       ]
     },
+    "AzureActiveDirectory": {
+      "description": "The settings to enable AAD authentication on the cluster",
+      "properties": {
+        "tenantId": {
+          "description": "Azure active directory tenant id",
+          "type": "string",
+          "minLength": 1
```

Comment:
minlength for string, does not look good,, remove it, applicable elsewhere like properties below.